torch/csrc/jit/operator_upgraders/version_map.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace torch {
+namespace jit {
+
+struct UpgraderEntry {
+  int version_bump;
+  std::string upgrader_name;
+  std::string old_schema;
+};
+
+static std::unordered_map<std::string, std::vector<UpgraderEntry>> operator_version_map(
+    {{"div.Tensor",
```

Comment:
How do you swap the operator in the graph? When I try to replace the operator in bytecode, the operator name will be `aten::div.Tensor`, instead of `div.Tensor`.

Reply:
good catch! It should be aten::div.Tensor

Reply:
Oh ok. Thanks. I manually add it in the codegen file in https://github.com/pytorch/pytorch/pull/69194. Will remove the manual part after this pr is updated.